article-analyzer/src/main/java/co/inventorsoft/academy/spring/service/ArticleAnalyzer.java

```diff
@@ -0,0 +1,25 @@
+package co.inventorsoft.academy.spring.service;
+
+import co.inventorsoft.academy.spring.repository.CategoryRepository;
+import co.inventorsoft.academy.spring.resolver.NotificationServiceResolver;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.springframework.stereotype.Service;
+
+@Data
```

Comment:
don't use @Data (@Getter, @Setter) on services classes, you don't use these

Reply:
Data only on model classes?

Reply:
yes, check what it is, https://projectlombok.org/features/Data

Reply:
@RequiredArgsConstructor is enough